.config/dotnet-tools.json

```diff
@@ -24,7 +24,7 @@
       "rollForward": false
     },
     "husky": {
-      "version": "0.7.2",
+      "version": "0.8.0",
```

Comment:
I know https://github.com/riok/mapperly/pull/2028 is doing this too, but I was unable to build or commit on my environment as I only have .net 10, and anything husky related was failing to work (i.e. entire build or commit hooks) because 0.7.2 is .net 9 only